tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "skipLibCheck": true, // do not check types in node_modules folder
-    "target": "esnext",
+    "target": "es2019",
```

Comment:
I much prefer the targeting of a specific year, rather than a generic "next" or "current".

According to this site - https://carloscaballero.io/es2020-features-in-simple-examples/ - nullish coalescing operators are in the ES2020 standard, not ES2019

Reply:
if you set it to es2020 it does comes up with errors
works with es2018, es2019
fails  with es2020, esnext